docs/claimantmodel/Book.md

```diff
@@ -0,0 +1,130 @@
+  - Human error in following process
+  - Massively unlikely, but [possible](https://www.agwa.name/blog/post/how_ct_logs_fail) random uncontrollable events
+- Greed: the Claimant decided that abusing their trust was financially more beneficial than continuing operating in
+  a trustworthy manner (e.g. exit scams in the cryptocurrency world)
+- Compromise:
+  - Insider Threat: a single agent at the company has motivation to exploit trust
+  - Leverage: a third-party could coerce the Claimant into betraying trust
+  - Compromise: actors that breach the Claimant security mechanisms could issue false claims directly by stealing signing keys,
+    or change code or playbooks in such a way that the Claimant themselves issues false claims
+
+There are also pressures in the opposite direction, i.e. motivations to *never* issue a bad claim.
+Consider the situation where each Claim is provided to a single Believer who would never realize the Claim was bad, and then
+compare this to a situation where every Claim is made in a publicly visible forum.
+In the first situation, maybe a Claimant could be coerced into making a bad Claim "just this one time", where in the second
+case it would be discovered and a public investigation launched.
+This is the motivation for [transparency logs](https://transparency.dev).
+
+## Verification
+
+Given enough time, potential motivation, and/or bad luck, any Claimant could issue a false Claim.
+The question is how to make a system with enough security given this fact.
+
+Earlier we established that Claims must be falsifiable, but that in general the Believer will not be able
+to perform this verification.
+Thus we introduce the role of the `Verifier`, which will be able to verify all Claims in a system.
+
+TODO: link to sidebar discussing roles vs actors: the Verifier role does not imply there is a single actor that must
+be able to verify all the claims.
+
+## More Precision
+
+In the interests of making the above as accessible to newcomers as possible, some amount of precision has
+been sacrificed.
+If you're still following along at this point, then it seems like a good time to discuss some topics that
+aren't necessarily difficult, but cross-cut the model in ways that make it too overwhelming to introduce earlier.
+
+### Statements vs Claims
+
+We have used the term Claim until this point to mean two related, but conceptually distinct things:
+- A falsifiable logical statement
+- Data
+
+The first of these is the correct usage of `Claim`: a falsifiable logical statement made by the Claimant.
+The second of these is often referred to as the Claim at high levels, but where precision is required we should refer
+to this as the `Statement`.
+
+A Statement must be issued by the Claimant and it must be possible to construct the Claim from it.
+In practical scalable systems, Statements are determined to have been issued by a Claimant by the presence of
+a digital signature over the Statement.
```

Comment:
A concrete example here, i.e TLS certificate?

Reply:
+1, one or more example that people can follow along in this doc, with details related to this example in each section would ease understanding (and ideally, an example without SCTs)

Reply:
Yep, added an example here.